README.md

```diff
@@ -1,38 +1,56 @@
 # Neural Network Compression Framework (NNCF)
 
-This repository contains a PyTorch\*-based framework and samples for neural networks compression.
+NNCF provides a suite of advanced compression algorithms, applied during a model fine-tuning and samples for models 
+compression. It is highly aligned with [OpenVINO&trade;](https://github.com/openvinotoolkit/openvino) and is aimed 
+to improve runtime performance of models in OpenVINO&trade; with minimal accuracy drop.
 
-The framework is organized as a Python\* package that can be built and used in a standalone mode.
-The framework architecture is unified to make it easy to add different compression methods.
+NNCF is designed to work with models from the two most popular deep learning frameworks, [PyTorch](https://pytorch.org/) 
+and [TensorFlow](https://www.tensorflow.org/).
 
-The samples demonstrate the usage of compression algorithms for three different use cases on public models and datasets: Image Classification, Object Detection and Semantic Segmentation.
-[Compression results](#nncf-compressed-model-zoo) achievable with the NNCF-powered samples can be found in a table at the end of this document.
+The framework is organized as a Python\* package that can be built and used in a standalone mode. The framework 
+architecture is unified to make it easy to add different compression algorithms for both PyTorch and TensorFlow deep 
+learning frameworks.
+
+The samples demonstrate the usage of compression algorithms for three different use cases on public PyTorch and 
+TensorFlow models and datasets: Image Classification, Object Detection and Semantic Segmentation. 
+[Compression results](#nncf-compressed-model-zoo) achievable with the NNCF-powered samples can be found in a table at 
+the end of this document.
 
 ## Key Features
 
-- Support of various compression algorithms, applied during a model fine-tuning process to achieve best compression parameters and accuracy:
-    - [Quantization](./docs/compression_algorithms/Quantization.md)
-    - [Binarization](./docs/compression_algorithms/Binarization.md)
-    - [Sparsity](./docs/compression_algorithms/Sparsity.md)
-    - [Filter pruning](./docs/compression_algorithms/Pruning.md)
-- Automatic, configurable model graph transformation to obtain the compressed model. The source model is wrapped by the custom class and additional compression-specific layers are inserted in the graph.
-- Common interface for compression methods
-- GPU-accelerated layers for faster compressed model fine-tuning
-- Distributed training support
+- Support of various compression algorithms, applied during a model fine-tuning process to achieve best compression 
+  parameters and accuracy:
```

Comment:
` ...to achieve best compression parameters and accuracy` is not fully clear and justified to me. Let's remove or rephrase, like, `...applied during a model fine-tuning process to achieve a better performance-accuracy trade-off.`

Reply:
Done